packages/p2p/src/components/my-ads/create-ad-form.jsx

```diff
@@ -97,6 +97,13 @@ const CreateAdForm = () => {
                 {({ errors, handleChange, isSubmitting, isValid, setFieldValue, touched, values }) => {
                     const is_sell_advert = values.type === buy_sell.SELL;
 
+                    const invalidAmountError = Object.values(errors).includes('Enter a valid amount');
+                    if (invalidAmountError) {
+                        Object.entries(errors).forEach(([key, value]) => {
+                            errors[key] = value === 'Enter a valid amount' ? value : undefined;
+                        });
+                    }
+
```

Comment:
why do we need this change?

Reply:
That changes was added automatically, don't know how. Should i remake the PR?